printable-recipe-generator/src/main.ts

```diff
@@ -3,8 +3,63 @@ import * as process from 'node:process';
 import path from 'path';
 import type { Data } from 'ejs';
 import { render as renderTemplate } from 'ejs';
+import fetch from 'node-fetch';
+
+//load Contributors
+async function getChefs(): Promise<unknown> {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call -- response is expected to be JSON, safe to assign
+	const resp = await fetch(
+		'https://recipes.guardianapis.com/v2/contributors.json',
```

Comment:
you should be fetching from CODE environment if this is running in CODE or PROD if it's running in PROD.  Check the environment variable setup in the real lambda - I believe that you should already have a STAGE var set up.

Reply:
Fair point, I will change this, but a question: Are we not having CODE and PROD aligned anyway for contributors list?

Reply:
No as it's possible that upstream changes could change the data format.... we'd want to be able to test this in CODE (there is a CODE tag manager and CAPI that the data sources from)